src/Ninja/G1CP/Content/Tests/test007.d

```diff
@@ -70,23 +65,17 @@ func int  ZS_G1CP_Test_007_NpcRountine_Loop() {
         G1CP_Test_007_Pass += 1;
 
         // Trigger the AI state
-        var int symbId; symbId = MEM_GetSymbolIndex("ZS_PracticeSword");
-        if (symbId != -1) {
-            // AI_StartState(self, symbId, 0, ""); // Does not work, expects func parameter
-            MEM_PushInstParam(self);
-            MEM_PushIntParam(symbId); // Func parameter as integer
-            MEM_PushIntParam(0);
-            MEM_PushStringParam("");
-            MEM_Call(AI_StartState);
+        // AI_StartState(self, symbId, 0, ""); // Does not work, expects func parameter
+        MEM_PushInstParam(self);
+        MEM_PushIntParam(MEM_GetSymbolIndex("ZS_PracticeSword")); // Func parameter as integer
```

Comment:
Wouldn't it be better to use `MEM_PushIntParam(funcId);` and save the return value of `G1CP_Testsuite_CheckAiStateStartFunc("ZS_PracticeSword");` in line 14?

Reply:
Would have been nice, but it's not within the same function.